geonode/harvesting/tasks.py

```diff
@@ -209,6 +209,8 @@ def update_harvestable_resources(self, harvester_id: int):
     worker = harvester.get_harvester_worker()
     try:
         num_resources = worker.get_num_available_resources()
+        harvester.num_harvestable_resources = num_resources
+        harvester.save()
```

Comment:
Please move these two lines to the `else` branch of this `try` block

Reply:
updated @ricardogsilva